web/themes/joinup/js/script.js

```diff
@@ -3,22 +3,48 @@
  * Joinup theme scripts.
  */
 
- jQuery(document).ready(function ($) {
-   $('#edit-delete').addClass('button--default button--blue mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent');
+(function ($) {
+  Drupal.behaviors.deleteButton = {
+    attach: function (context, settings) {
+      $(context).find('#edit-delete').once('deleteButton').each(function () {
+        $(this).addClass('button--default button--blue mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent');
+      });
+    }
+  };
+
+  Drupal.behaviors.silderSelect = {
+    attach: function (context, settings) {
+      $(context).find('.slider__select').once('sliderSelect').each(function () {
+         var select = $(this);
+         var selectLength = $(this).find('option').length;
```

Comment:
Instead of `$(this)` use `select` variable to avoid creating again another jQuery object.
If possible and we all agree, I like more to put a dollar sign in front of variables that are jQuery objects. So it's easy to understand what is inside.